docs/faq.rst

```diff
@@ -258,9 +258,11 @@ Postfix, Dovecot, Nginx and Rspamd support overriding configuration files. Overr
 correct syntax. The following file names will be taken as override configuration:
 
 - `Postfix`_ :
-   - ``main.cf`` as ``$ROOT/overrides/postfix/postfix.cf``
-   - ``master.cf`` as ``$ROOT/overrides/postfix/postfix.master``
+   - ``main.cf`` as ``$ROOT/overrides/postfix.cf``
+   - ``master.cf`` as ``$ROOT/overrides/postfix.master``
```

Comment:
The actual behaviour is that the overrides reside in $ROOT/overrides/postfix/postfix.master.
This was changed in mailu 1.8.